src/microdns.pc.in

```diff
@@ -7,5 +7,4 @@ Name: microDNS
 Description: mDNS simple implementation
 Version: @VERSION@
 Libs: -L${libdir} -lmicrodns @LIBSOCKET@
-Libs.private: @LIBS@
 Cflags: -I${includedir}
```

Comment:
Actually, can't we use the [pkgconfig module](https://mesonbuild.com/Pkgconfig-module.html) for this?

Reply:
we can, but I'm doing a straight port here :)